README.md

```diff
@@ -10,6 +10,31 @@ The majority of documentation on the site is drawn from [the probot docs](https:
 
 We also use this website to show off some of the cool probot plugins built by the community. If you would like to have your plugin listed there, check out our [Contributing Guidelines](https://github.com/probot/probot.github.io/blob/master/CONTRIBUTING.md) for more information about what the requirements are to do that.
 
+## Install and Usage
+
+If you'd like to work on this repository, clone it. We use [Jekyll](https://jekyllrb.com/) to build the website, hosted on [GitHub Pages](https://pages.github.com/). We also use Git submodules to pull docs from the docs folder in  [probot/probot](https://github.com/probot/probot/tree/master/docs).
+
+Once you've cloned this repo, run these commands in this directory:
+
+```sh
+# Only needed the first time:
+script/bootstrap
+# And each time you work locally:
+script/server
+```
+
+You should then be able to navigate to the URL printed in your console, and see live edits you make render in your browser. You'll likely have to familiarize yourself with how Jekyll works to do larger edits, but it's worth the effort!
+
+Another note: You can see more commands we use, such as our native build and server functions, in `script`. Go take a look.
```

Comment:
some nits:
"in the `script` folder"

I may overuse `!`, but how about "Go take a look!" and add a link to the script folder on GitHub? (https://github.com/probot/probot.github.io/tree/master/script)

Reply:
Sounds good to me. I'll add this.